templates/web/base/questionnaire/index.html

```diff
@@ -1,18 +1,37 @@
 [%
-    pre_map = INCLUDE 'report/_main.html';
+    SET bodyclass = 'mappage';
+    PROCESS "report/photo-js.html";
     PROCESS "maps/${map.type}.html";
-    INCLUDE 'header.html', title = loc('Questionnaire')
+    INCLUDE 'header.html', title = loc('Questionnaire');
 %]
 
 [% map_html %]
 
-[% INCLUDE 'report/updates.html' %]
-
 </div>
-<div id="side">
 
 <h1>[% loc('Questionnaire') %]</h1>
 
+<h2 class="questionnaire-report-header">Your report</h2>
+<div class="questionnaire-report-reminder">
+    [% INCLUDE 'report/photo.html' object=problem %]
+    <h3 class="questionnaire-report-reminder__report-title">
+        <a href="/report/[% problem.id %]">[% problem.title | html %]</a>
+    </h3>
+    <p class="questionnaire-report-reminder__report-meta">[% problem.meta_line(c) | html %]</p>
+  [% IF updates.size %]
+    <p class="questionnaire-report-reminder__last-update-header">
+        <strong>Last update</strong>
+        <a href="/report/[% problem.id %]">Show all updates</a>
+    </p>
+    <p class="questionnaire-report-reminder__last-update">&ldquo;[% add_links( updates.last.text ) %]&rdquo;</p>
+  [% END %]
+</div>
+
+<p>
+[% loc('An update marked this problem as fixed.') IF problem.is_fixed %]
+[% loc('Has this problem been fixed?') %]
+</p>
```

Comment:
Is there a reason the error output, if any, appears between this question and its possible answers? Could this `p` be moved down to next to the answers?